plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinates.java

```diff
@@ -95,8 +94,8 @@ public String getArtifactId() {
 
   @Override
   public String toString() {
-    return MessageFormat.format("MavenCoordinates [repository={0}, {1}:{2}:{3}:{4}:{5}]",
-                                repository, groupId, artifactId, type, classifier, version);
+    return String.format("MavenCoordinates [repository={0}, {1}:{2}:{3}:{4}:{5}]", repository,
```

Comment:
`String.format()` isn't equivalent to `MessageFormat.format()`.

Reply:
reverted